libcloud/loadbalancer/base.py

```diff
@@ -222,6 +222,19 @@ def get_balancer(self, balancer_id):
         raise NotImplementedError(
             'get_balancer not implemented for this driver')
 
+    def get_tags(self, balancer_id):
```

Comment:
would be better named `list_balancer_tags`, to be more consistent with the other base methods.